<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PythonREPL</class>
 <widget class="QMainWindow" name="PythonREPL">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CloudCompare - Python REPL</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../CloudCompare-PythonRuntime.qrc">
    <normaloff>:/CC/plugin/PythonRuntime/images/repl-icon.png</normaloff>:/CC/plugin/PythonRuntime/images/repl-icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QListWidget" name="outputDisplay"/>
      <widget class="QPlainTextEdit" name="codeEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>20</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="resetAction"/>
  </widget>
  <action name="resetAction">
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>Reset the REPL</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../CloudCompare-PythonRuntime.qrc"/>
 </resources>
 <connections/>
</ui>
